gradle/libs.versions.toml

```diff
@@ -0,0 +1,23 @@
+[versions]
+kotlinVersion = "1.5.10"
+androidAppCompatVersion = "1.2.0"
+materialDesignVersion = "1.0.0"
+androidLifecycleVersion = "2.1.0"
+androidCoreTestingVersion = "2.1.0"
+coroutinesVersion = "1.4.3"
```

Comment:
Update coroutines to latest version https://github.com/Kotlin/kotlinx.coroutines/releases/tag/1.5.0